general/quick-start.md

```diff
@@ -17,7 +17,7 @@ title: Quick Start
 4. Secure the server with a method of your choice.
 
     * Create an SSL certificate and add it on the **Networking** page.
-    * Put your server behind a [reverse proxy](xref:admin-reverse-proxy).
+    * Put your server behind a [reverse proxy](networking/index.md#running-jellyfin-behind-a-reverse-proxy).
```

Comment:
I couldn't get the `xref` version of this link to work, but based on the [DocFx documentation](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#using-cross-reference) it looks like a relative link should be preferred in this case anyway